src/physics/utils/tropopause_climo_read.F90

```diff
@@ -122,6 +122,7 @@ subroutine tropopause_climo_read_file()
     real(kind_phys), parameter :: d2r=pi/180._kind_phys, zero=0._kind_phys, twopi=pi*2._kind_phys
     character(len=shr_kind_cl) :: locfn
     character(len=shr_kind_cm) :: errmsg
```

Comment:
would it make sense to change this errmsg to "shr_kind_cl" as well to match the other error messages? It's probably unlikely to be too short, but would be consistent with the other usage.